main.go

```diff
@@ -80,13 +82,15 @@ func run() error {
 	if err != nil {
 		return fmt.Errorf("unable to create client: %v", err)
 	}
-	imageClient, err := imageclientset.NewForConfig(config)
+	// Config and Client to access release images
+	releaseConfig, err := loadKubeconfigFromFlagOrDefault(opt.ReleaseClusterKubeconfig, prowJobKubeconfig)
```

Comment:
In order to keep backwards compatibility, if opt.ReleaseClusterKubeConfig is empty it should use BuildClusterKubeconfig.

Reply:
Updated code.